src/N3Lexer.js

```diff
@@ -463,6 +463,11 @@ export default class N3Lexer {
 
   // ## Public methods
 
+  // ### `setComments` enables or disables creating a token per comment using a boolean.
+  setComments(enable) {
+    this._comments = enable;
+  }
+
```

Comment:
FYI, JS can do setters by just

```js
{
//...
  set comments(value) {
    this._comments = value;
  }
//...
}
```

but in this case, we can probably just rename `_comments` into `comments` to make it public.

Reply:
Just making it public makes sense. Done that